test/test_expr.jl

```diff
@@ -451,4 +451,70 @@ function test_quadexpr_owner_model()
     return
 end
 
+function test_aff_expr_complex_lower_bound()
+    model = Model()
+    @variable(model, x in ComplexPlane())
+    y = real(x)
+    @test !has_lower_bound(y)
+    set_lower_bound(y, 1)
+    @test has_lower_bound(y)
+    @test lower_bound(y) == 1
+    delete_lower_bound(y)
+    @test !has_lower_bound(y)
+    return
+end
+
+function test_aff_expr_complex_upper_bound()
+    model = Model()
+    @variable(model, x in ComplexPlane())
+    y = real(x)
+    @test !has_upper_bound(y)
+    set_upper_bound(y, 1)
+    @test has_upper_bound(y)
+    @test upper_bound(y) == 1
+    delete_upper_bound(y)
+    @test !has_upper_bound(y)
+    return
+end
+
+function test_aff_expr_complex_start_value()
+    model = Model()
+    @variable(model, x in ComplexPlane())
+    y = real(x)
+    @test start_value(y) === nothing
+    set_start_value(y, 1)
+    @test start_value(y) == 1
+    return
+end
+
+function test_aff_expr_complex_error()
+    model = Model()
+    @variable(model, x[1:2, 1:2] in HermitianPSDCone())
+    @test_throws(
+        ErrorException(
+            "Cannot call $start_value with $(x[1, 1]) because it is not a real-valued affine " *
+            "expression of one variable with a coefficient of `+1`. Use " *
+            "`real(x)` or `imag(x)` to obtain the real and imaginary part and " *
+            "pass that instead.",
+        ),
+        start_value(x[1, 1]),
```

Comment:
It may seem a bit weird that this does not work since the diagonal of a hermitian matrix is real